fedora_messaging/tests/unit/test_message.py

```diff
@@ -447,12 +447,6 @@ def test_body_deprecation_getter(self, mock_warning):
             stacklevel=2,
         )
 
-    def test_topic_prefix(self):
```

Comment:
It seems like this test can stay, it just should check that the msg._encoded_routing_key is "prefix.test.topic".encode("utf-8")